services/rag/Dockerfile

```
@@ -13,11 +13,13 @@ WORKDIR /app
 # - curl: Health checks and downloads
 # - build-essential: Compilation for native Python packages
 # - libpq-dev: PostgreSQL client library for asyncpg
+ARG SOPS_VERSION=3.9.4
 RUN apt-get update && apt-get install -y --no-install-recommends \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)